pages/apis/graphql/graphql_cookbook.md

```diff
@@ -350,6 +350,20 @@ mutation UpdateSessionDuration {
 }
 ```
 
+## Pin SSO sessions to IP addresses
+
+You can require users to re-authenticate with your SSO provider when their IP address changes with the following call, replacing `ID` with the GraphQL ID of the SSO Provider:
```

Comment:
Nitpicks:

- I don't think the "GraphQL" is needed.
- Provider doesn't need to be capitalized here.

```suggestion
You can require users to re-authenticate with your SSO provider when their IP address changes with the following call, replacing `ID` with the ID of the SSO provider:
```

Reply:
@mbelton-buildkite there's a few different ID's in our GraphQL. Leaving `GraphQL` in there to disambiguate from UUID or the model id.

Reply:
👍🏻 Sounds good